WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/ShippingLabelPackageList.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+import Yosemite
+
+struct ShippingLabelPackageList: View {
+    @EnvironmentObject private var viewModel: ShippingLabelPackageDetailsViewModel
+    @Environment(\.presentationMode) var presentation
+
+    var body: some View {
+        ScrollView {
+            VStack(spacing: 0) {
+
+                /// Custom Packages
+                ///
+                if viewModel.customPackages.count > 0 {
+                    ListHeaderView(text: Localization.customPackageHeader.uppercased(), alignment: .left)
+                        .background(Color(.listBackground))
+                }
+                ForEach(viewModel.customPackages, id: \.title) { package in
+                    let selected = package == viewModel.selectedCustomPackage
+                    SelectableItemRow(title: package.title, subtitle: package.dimensions + " \(viewModel.dimensionUnit)", selected: selected).onTapGesture {
+                        viewModel.didSelectPackage(package.title)
+                    }
+                    Divider().padding(.leading, Constants.dividerPadding)
+                }
+
+                /// Predefined Packages
+                ///
+                ForEach(viewModel.predefinedOptions, id: \.title) { option in
+
+                    ListHeaderView(text: option.title.uppercased(), alignment: .left)
+                        .background(Color(.listBackground))
+                    ForEach(option.predefinedPackages, id: \.id) { package in
+                        let selected = package == viewModel.selectedPredefinedPackage
```

Comment:
Since we already have an `id`, probably a good idea to conform `predefinedPackages` objects into `Identifiable`, right?

Reply:
Good idea 👍  9a7d0b9964e5b8c3870486464af71927afc81f23